build.gradle

```diff
@@ -135,7 +146,9 @@ subprojects {
             compile deps.log4j
             compile deps.guice
             compile deps.slf4j
-
+            // See: https://github.com/rzwitserloot/lombok/issues/1301
+            // See: https://github.com/tbroyer/gradle-errorprone-plugin#usage
+            errorprone 'com.google.errorprone:error_prone_core:2.0.15'
```

Comment:
I specified the errorprone version recently too. Your way of specifying the dependency seems to be simpler.  How did you verify the specified version was used?

Reply:
Until I got this version, I was not able to build azkaban with gobblin metrics applied. Setting to this version solved this issue. I cross checked my local cache and could see this version.